packages/embark-code-generator/src/index.js

```diff
@@ -422,11 +422,15 @@ class CodeGenerator {
   }
 
   buildContractJS(contractName, contractJSON, cb) {
-    let contractCode = "import EmbarkJS from '../embarkjs';\n";
-    contractCode += `let ${contractName}JSONConfig = ${JSON.stringify(contractJSON)};\n`;
-    contractCode += `let ${contractName} = new EmbarkJS.Blockchain.Contract(${contractName}JSONConfig);\n`;
-
-    contractCode += "export default " + contractName + ";\n";
+    const contractCode = `
+      const isNodeJS = (typeof process !== 'undefined' && process.versions && process.versions.node);
+      const lib = isNodeJS ? '../embarkjs.node' : '../embarkjs';
+      const EmbarkJS = require(lib);
+
+      let ${contractName}JSONConfig = ${JSON.stringify(contractJSON)};
+      let ${contractName} = new EmbarkJS.Blockchain.Contract(${contractName}JSONConfig);
+      export default ${contractName};
```

Comment:
I think this is where problems might arise. `export` doesn't exist in Node. I forgot to mention it to you because I hadn't notice because the problem the user was having was with the import.
When trying to do the node embarkjs, it's the export that gave me the most problems, because you can't put export in an `if`... Maybe you can find a solution, but maybe we'll need to transpile those too 😞

Reply:
Wait, does `module.exports` not work on the browser?

Reply:
Last I tried, it didn't. Maybe it was something I was doing wrong.